src/igmpproxy.h

```diff
@@ -155,7 +155,13 @@ struct IfDesc {
     unsigned int        robustness;
     unsigned char       threshold;   /* ttl limit */
     unsigned int        ratelimit;
-    unsigned int        index;
+    int                 index;
```

Comment:
Looking at those whole code and I think it does not make sense to change type of index from unsigned to signed. On lot of places in this pull request is changed index type from unsigned to signed integer, but all other places (e.g. getIfByVifIndex) can accept only valid interface index and therefore non-negative value -- which perfectly fits into unsigned type (which was used before).

Signed integer is used only because special value `-1` represents invalid value. But for this one case, special construct `(unsigned int)-1` is enough. We do not need to represent more other negative states.

Reply:
you are right, it's back to unsigned now